homeassistant/components/homekit/type_sensors.py

```diff
@@ -75,3 +84,53 @@ def update_state(self, entity_id=None, old_state=None, new_state=None):
             self.char_humidity.set_value(humidity, should_callback=False)
             _LOGGER.debug('%s: Percent set to %d%%',
                           self.entity_id, humidity)
+
+
+@TYPES.register('BinarySensor')
+class BinarySensor(HomeAccessory):
+    """Generate a BinarySensor accessory as binary sensor."""
+
+    def __init__(self, hass, entity_id, name, attributes, **kwargs):
+        """Initialize a BinarySensor accessory object."""
+        super().__init__(name, entity_id, CATEGORY_SENSOR, **kwargs)
+
+        self.hass = hass
+        self.entity_id = entity_id
+
+        service_map = {
```

Comment:
Please move this between `_LOGGER` and `@TYPES.register('TemperatureSensor')` and rename it to something like `BINARY_SENSOR_SERVICE_MAP`

Reply:
Fixed